plugins/catalog-backend/src/catalog/types.ts

```diff
@@ -50,18 +51,23 @@ export type EntitiesSearchFilter = {
   /**
    * Match on plain equality of values.
    *
-   * If undefined, this factor is not taken into account. Otherwise, match on
-   * values that are equal to any of the given array items. Matches are always
-   * case insensitive.
+   * Match on values that are equal to any of the given array items. Matches are
+   * always case insensitive.
    */
-  matchValueIn?: string[];
+  values: string[];
```

Comment:
With the `not` in place this is actually really similar to the existing code I realized 😅 

I think the rename from `matchValueIn` to `values` still makes sense though, especially if this ends up as a more public API. Def makes sense to get rid of `matchValueExists`.

I do wonder if this is a bit of a premature split into two types and that we should just keep this in place for now?

```ts
export type EntitiesSearchFilter = {
  key: string
  values?: string[]
}
```

It does end up being two fewer exported symbols to worry about

Reply:
I'm fine with this! Will make the change.